sample_inventories/rbac_mtls_certificates_example.yml

```diff
@@ -0,0 +1,132 @@
+### RBAC MTLS User provided certificates example 
+##
+## The following is an example inventory file of the configuration required for setting up a kafka cluster with RBAC mTLS enabled.  We have removed all comments from
+## the standard inventory example to help with understanding the formatting.
+
+all:
+  vars:
+    ansible_connection: ssh
```

Comment:
update these sample files, they are outdated, samples of vars can be found in molecule.yml files